packages/taco/src/index.ts

```diff
@@ -12,3 +12,12 @@ export {
 export * as conditions from './conditions';
 
 export { decrypt, encrypt, encryptWithPublicKey } from './taco';
+
+export { type ValidationResult } from './config-validator';
```

Comment:
Is this needed externally in the API? I thought it was only used internally for `TacoClient.validateConfig`  which either returns nothing, or an Error?